br/pitr-known-issues.md

```diff
@@ -53,3 +56,23 @@ Issue 链接：[#13030](https://github.com/tikv/tikv/issues/13030)
 Issue 链接：[#13126](https://github.com/tikv/tikv/issues/13126)
 
 在集群出现网络分区故障后，备份任务难以继续备份日志，并且在超过一定的重试时间后，任务会被置为 `ERROR` 状态。此时备份任务已经停止，需要手动执行 `br log resume` 命令来恢复日志备份任务。
+
+## 集群监控显示数据增量 385 GB，日志备份实际消耗存储大约 1 TB
+
+Issue 链接：[#13306](https://github.com/tikv/tikv/issues/13306)
+
+这是由于集群监控显示的是 rocksdb 压缩后的数据，而日志备份使用的是自定义的编码方式存储 KV 数据，因此造成压缩比率不一致，大约是 2～3 倍之间。而日志备份没有选择使用 rocksdb 生成 SST 文件的方式存储数据的原因是，日志备份期间生成的数据会遇到区间范围过大，但是实际区间内容较少的情况，这个情况下，通过 ingest SST 的方式恢复数据，效果会大打折扣。
+
+## 执行 PITR 恢复时遇到 `execute over region id` 报错
+
+Issue 链接：[#37207](https://github.com/pingcap/tidb/issues/37207)
+
+该场景发生在通过日志数据来恢复全量导入的集群初始化数据，经过测试发现，当存在长时间(24h)大量热点写入，且平均单台 TiKV 节点写入 ops > 50k/s(可以通过 grafana 中 `TiKV-Details` -> `Backup Log` -> `Handle Event Rate` 确认该数值)，那么有几率会遇到这个情况。
```

Comment:
```suggestion
该场景发生在全量数据导入时开启了日志备份，并使用 PiTR 恢复全量导入时间段的日志。经过测试发现，当存在长时间(24h)大量热点写入，且平均单台 TiKV 节点写入 ops > 50k/s(可以通过 grafana 中 `TiKV-Details` -> `Backup Log` -> `Handle Event Rate` 确认该数值)，那么有几率会遇到这个情况。
```

Reply:
I feel like both description is ok? WDYT? cc @IANTHEREAL